Packs/SafeBreach/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "## FILL OUT MANUALLY ##",
```

Comment:
please all the fields here properly
change the author, and url and email

Reply:
What we should put in useCases, tags and keywords?